tools/oeedger8r/Emitter.ml

```diff
@@ -891,6 +891,20 @@ let warn_signed_size_or_count_types (fd:Ast.func_decl) =
         | _ -> ()
     ) fd.Ast.plist
 
+let warn_size_and_count_params (fd:Ast.func_decl) =
+  let print_size_and_count_warning { ps_size; ps_count } =
+    match ps_size, ps_count with
+        | Some (Ast.AString p), Some (Ast.AString q) ->
+          printf "Warning: Function '%s': simultaneous 'size' and 'count' parameters '%s' and '%s' are not supported by oeedger8r.\n" fd.fname p q
```

Comment:
Since we don't support it it ought to be an error. I am not sure what the edger8r will pick up - count or size.

Reply:
Fixed.